--- a/os_ken/services/protocols/bgp/operator/ssh.py
+++ b/os_ken/services/protocols/bgp/operator/ssh.py
@@ -25,7 +25,6 @@
 
 import paramiko
 
-from os_ken import version
 from os_ken.lib import hub
 from os_ken.services.protocols.bgp.base import Activity
 from os_ken.services.protocols.bgp.operator.command import Command
@@ -59,7 +58,7 @@
 class SshServer(paramiko.ServerInterface):
     TERM = "ansi"
     PROMPT = "bgpd> "
-    WELCOME = "\n\rHello, this is OSKen BGP speaker (version %s).\n\r" % version
+    WELCOME = "\n\rHello, this is OSKen BGP speaker.\n\r"
 
     class HelpCmd(Command):
         help_msg = 'show this help'